src/utils/log/FeatureLog.ts

```diff
@@ -4,6 +4,7 @@ enum EFeatureLogFlag {
   // Hooks
   useSimpleWebsocketHook = 'useSimpleWebsocketHook',
   useOneSignalSubscribeToCommunityHooks = 'useOneSignalSubscribeToCommunityHooks',
+  useSystemMessage = 'useSystemMessage',
```

Comment:
<img src="https://avatars.githubusercontent.com/in/347564?s=41" alt="Image description" width="20" height="20">   CodeRabbit

The addition of `useSystemMessage` to the `EFeatureLogFlag` enum is a good way to extend the logging capabilities. This change seems appropriate and doesn't introduce any issues.
```diff
+  useSystemMessage = 'useSystemMessage',
```


<!-- This is an auto-generated comment by OSS CodeRabbit -->